etc/hosts.tmpl

```diff
@@ -7,7 +7,7 @@
 {{- if $netdev.Default.GetB}}
 {{$netdev.Ipaddr.Get}} {{$node.Id.Get}}
 {{- else}}
-{{$netdev.Ipaddr.Get}} {{$node.Id.Get}}-{{$devname}}
+{{$netdev.Ipaddr.Get}} {{$node.Id.Get}} {{$node.Id.Get}}-{{$netdev.Device.Get}}
```

Comment:
If there are multiple network devices, this will cause redundant entries for any given nodename.

Reply:
Talked with Greg and this is what we want in the /etc/hosts file for two network devices on a node:

n0000 is only on the default interface. n0000-default needs to be there.

```
10.0.8.150 n0000 n0000-default n0000-eth0
10.0.16.150 n0000-infiniband n0000-ib0
```